app/lib/package/model_properties.dart

```diff
@@ -107,7 +107,6 @@ class Pubspec {
   SdkConstraintStatus get _sdkConstraintStatus =>
       SdkConstraintStatus.fromSdkVersion(_inner.environment?['sdk'], name);
 
-  // TODO: migrate uses to SdkConstraintStatus.isDart2Compatible
```

Comment:
Did we do this, or we no longer want to?

Reply:
Oh, that's an unrelated change I shall take it out from the PR. I was looking at TODOs in the code, and I picked it out first, as it seemed to be obsolete.